src/Unit-2/lesson5/README.md

```diff
@@ -0,0 +1,150 @@
+# Unit 2, Lesson 5 - The controversial `LoadDocument`
```

Comment:
This is a bad title. It warns you about a feature, but we are sending this to customers.
Why send them an lesson about a feature that we tell them not to use?